packages/mock/src/data/match.ts

```diff
@@ -1,39 +1,57 @@
 import { RunResult } from "./runResult";
 import { teams } from "./team";
 
-export interface Match {
+export interface preMatch {
```

Comment:
元が`Match`なので先頭大文字は揃えた方が良さそう
```suggestion
export interface PreMatch {
```